<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccStartWnd</class>
 <widget class="QWidget" name="AccStartWnd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1135</width>
    <height>790</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>1091</width>
     <height>591</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Nirmala UI&quot;;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_back">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_acc">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>750</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Пользователь: </string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_map">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>660</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Показать на карте</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_resetfilter">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>10</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Сброс</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEditfrom">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>10</y>
     <width>110</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit_to">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>10</y>
     <width>110</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_datefrom">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>10</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Дата регистрации с</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_applyfilter">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>10</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Применить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_showfilter">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Фильтр по дате</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_dateto">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>10</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>по</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_add">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>660</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Добавить ДТП</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
